programs/adapter-tulip/src/lib.rs

```diff
@@ -116,29 +116,25 @@ pub mod adapter_tulip {
                 withdraw_deposit_tracking_data.append(&mut input_struct.farm_type_0.try_to_vec()?);
                 withdraw_deposit_tracking_data.append(&mut input_struct.farm_type_1.try_to_vec()?);
 
-                let accounts = load_remaining_accounts(
+                let withdraw_deposit_tracking_index_array = vec![0, 15, 16, 20, 21, 22, 12, 14, 1];
+                let withdraw_deposit_tracking_accounts = load_remaining_accounts(
                     ctx.remaining_accounts,
-                    vec![0, 15, 16, 20, 21, 22, 12, 14, 1],
+                    withdraw_deposit_tracking_index_array.clone(),
                 );
 
                 let withdraw_deposit_tracking_ix = Instruction {
                     program_id: ctx.accounts.base_program_id.key(),
-                    accounts,
+                    accounts: withdraw_deposit_tracking_accounts,
                     data: withdraw_deposit_tracking_data,
                 };
+                let withdraw_deposit_tracking_account_infos = account_info_array(
+                    ctx.remaining_accounts,
+                    withdraw_deposit_tracking_index_array.clone(),
+                );
                 invoke(
                     &withdraw_deposit_tracking_ix,
-                    &[
-                        ctx.remaining_accounts[0].clone(),
-                        ctx.remaining_accounts[15].clone(),
-                        ctx.remaining_accounts[16].clone(),
-                        ctx.remaining_accounts[20].clone(),
-                        ctx.remaining_accounts[21].clone(),
-                        ctx.remaining_accounts[22].clone(),
-                        ctx.remaining_accounts[12].clone(),
-                        ctx.remaining_accounts[14].clone(),
-                        ctx.remaining_accounts[1].clone(),
-                    ],
+                    &withdraw_deposit_tracking_account_infos
+                        [0..withdraw_deposit_tracking_index_array.len()],
```

Comment:
Move slice part to `account_info_array`

Reply:
Do you mean this?
```
let withdraw_deposit_tracking_account_infos = 
    get_account_info_array(...)[0..withdraw_deposit_tracking_index_array.len()];
```
if so this doesn't works, cause it might have the static size issue too.
-> the size for values of type `[anchor_lang::prelude::AccountInfo<'_>]` cannot be known at compilation time